guides/common/modules/con_kickstart-workflow.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Kickstart-Workflow_{context}"]
+= Kickstart Workflow
+
+You can use Kickstart to automate the installation process of a {ProjectName} or {SmartProxyServer} by creating a Kickstart file that contains all the information that is required for the installation.
+
+When you run a {ProjectName} Kickstart script, the following workflow occurs:
```

Comment:
How about this? It would make it clearer what "it" is.

```suggestion
When you run a {ProjectName} Kickstart script, the script performs the following actions:
```

Reply:
With this wording, we might even omit the "It" from each bullet, which would make it more scannable.